libafl/src/inputs/bytes.rs

```diff
@@ -48,6 +51,13 @@ impl Input for BytesInput {
         file.read_to_end(&mut bytes)?;
         Ok(BytesInput::new(bytes))
     }
+
+    /// Retrieve a unique name for this input
+    fn unique_name(&self) -> String {
```

Comment:
I'd say, maybe you should rename it `generate_name(idx: usize)` as we don't want necessary to ensure that is unique

Reply:
where `idx` is the `self.entries.len()`?

Reply:
Done.